huntsman/utils/hub_utils.py

```diff
@@ -0,0 +1,219 @@
+#A Python interface for the acroname USB Hubs - tested and functional
```

Comment:
This file needs some pep8 linting run on it, which will make it conform to style best-practices. If you are using Atom you can use this [package](https://atom.io/packages/pep8). Also see some of our text in the [CONTRIBUTING](https://github.com/panoptes/POCS/blob/e390b9990406f586d2be40b01ae1d7c5b86dd53d/CONTRIBUTING.md#code-formatting) doc.

Reply:
Fixed similarly to hue lights